### PR TITLE
Fixes: Only upload 1 rpi deb package

### PR DESCRIPTION
**Pull request by kminehart**
Resolves:

* https://github.com/grafana/grafana-build/issues/140
* Issue where the tar.gz was included in the deb installer, doubling its size.
* Issue where we were creating deb packages for rpi that weren't called "grafana-rpi" or "grafana-enterprise-rpi"

Summary after this is implemented:

* We will continue producing a deb package for armhf with the metadata name grafana and grafana-enterprise.
* We will start producing a deb package for armhf with the metadata name grafana-rpi or grafana-enterprise-pri as well.
* We won't be including the entire source grafana.tar.gz in the deb packages :doh:
* We won't be including storybook in those deb packages either.

### CODE REVIEW

File: scripts/move_packages_deb_test.go [hunk 1]
```diff
@@ -21,14 +21,12 @@ var debMapping = []m{
 	{
 		input: "gs://bucket/tag/grafana_v1.2.3_102_linux_arm-7.deb",
 		output: []string{
-			"artifacts/downloads/v1.2.3/oss/release/grafana_1.2.3_armhf.deb",
```
[the remark this answers is not in the document]
Comment (kminehart) [reply scripts/move_packages_deb_test.go:24]: yea that's correct.

interesting. i'll put it back then. good call out.

[the remark this answers is not in the document]
Comment (kminehart) [reply scripts/move_packages_deb_test.go:24]: that does complicate things a little bit hahahaha

[the remark this answers is not in the document]
Comment (zerok) [reply scripts/move_packages_deb_test.go:24]: And they also act as documentation to some degree <3

File: pipelines/package_names_test.go [hunk 2]
```diff
@@ -135,4 +149,50 @@ func TestOptsFromFile(t *testing.T) {
...
+		o := pipelines.TarFileOpts{
+			Name:    v[0],
+			Edition: v[1],
+			Version: "v1.0.0",
+			BuildID: "101",
+			// Edition is the flavor text after "grafana-", like "enterprise".
+			Distro: executil.Distribution("linux/amd64"),
+		}
+
+		if n := o.FullName(); n != fullname {
```
Comment (guicaulada) [inline pipelines/package_names_test.go:194]: I don't find this FullName function anywhere else but here, it's not defined or called in any other file.

File: scripts/drone_publish_tag_enterprise.sh [hunk 3]
```diff
@@ -75,4 +81,4 @@ dagger run --silent go run ./cmd docker \
 cat debs.txt rpms.txt zips.txt exes.txt docker.txt >> assets.txt
 
 # Move the tar.gz packages to their expected locations
-cat assets.txt | go run ./scripts/move_packages.go ./dist/prerelease
+# cat assets.txt | go run ./scripts/move_packages.go ./dist/prerelease
```
Comment (guicaulada) [inline scripts/drone_publish_tag_enterprise.sh:84]: Is move_packages not necessary on this script?

**Review by guicaulada — CHANGES_REQUESTED** [review]

**Review by guicaulada — APPROVED** [review]

**Pull request merged**